servers/rendering/shader_language.cpp

```diff
@@ -9717,6 +9717,25 @@ String ShaderLanguage::get_shader_type(const String &p_code) {
 	return String();
 }
 
+bool ShaderLanguage::is_builtin_func_out_parameter(const String &p_name, int p_param) {
+	int i = 0;
+	while (builtin_func_out_args[i].name) {
+		if (p_name == builtin_func_out_args[i].name) {
+			for (int j = 0; j < BuiltinFuncOutArgs::MAX_ARGS; j++) {
+				int arg = builtin_func_out_args[i].arguments[j];
+				if (arg == p_param) {
+					return true;
+				}
+				if (arg < 0) {
+					return false;
+				}
+			}
+		}
+		++i;
```

Comment:
As a matter of style, I prefer i++ and here they are equivalent.

Reply:
Hm, and I prefer maximum reasonable solutions (post-increment is not useful here)... Also interesting discussion about it - https://stackoverflow.com/questions/4706199/post-increment-and-pre-increment-within-a-for-loop-produce-same-output

But if you want I can fix it of course...

Reply:
Its not a big deal either way